includes/polling/os/apc.inc.php

```diff
@@ -1,94 +1,20 @@
 <?php
 
-// PDU
-$serial = trim(snmp_get($device, 'PowerNet-MIB::rPDUIdentSerialNumber.0', '-OQv'), '"');
+$apc_serial = snmp_get_multi_oid($device, 'rPDUIdentSerialNumber.0 atsIdentSerialNumber.0 upsAdvIdentSerialNumber.0 sPDUIdentSerialNumber.0 airIRRCUnitIdentSerialNumber.0 isxModularPduIdentSerialNumber.0 airIRRP100UnitIdentSerialNumber.0 airIRRP500UnitIdentSerialNumber.0', '-OUQs', 'PowerNet-MIB');
```

Comment:
I've tested you PR on an production environnement and here are the missing OID for an APC Netbotz 200 :

- $apc_serial : you can add emsIdentSerialNumber.0
- $apc_model : you can add emsIdentProductNumber.0
- $apc_hardware : you can add emsIdentHardwareRev.0
- $apc_version : you can add emsIdentFirmwareRev.0

Regards.